playground/ssr-vue/vite.config.js

```diff
@@ -85,14 +85,14 @@ export default defineConfig(({ command, ssrBuild }) => ({
           }
         },
         transform(code, id) {
+          const cleanId = cleanUrl(id)
           if (
             config.build.ssr &&
-            cleanUrl(id).endsWith('.js') &&
-            !code.includes('__ssr_vue_processAssetPath')
+            (cleanId.endsWith('.js') || cleanId.endsWith('.vue'))
           ) {
             return {
               code:
-                `import { __ssr_vue_processAssetPath } from '${virtualId}';` +
+                `import { __ssr_vue_processAssetPath } from '${virtualId}';__ssr_vue_processAssetPath;` +
```

Comment:
Why is this needed? And why the guard against importing it twice has been removed?

Reply:
Maybe `moduleSideEffects: 'no-treeshake'` means the module is side-effectful but the imports can be removed if not used?

Reply:
I would think that should be respected, but we could move with your change for the moment until we find out a better way.